packages/r3f/src/utils.ts

```diff
@@ -1,3 +1,7 @@
 import {useEditorStore} from './store'
+import type {ISheet} from '@theatre/core'
 
 export const refreshSnapshot = useEditorStore.getState().createSnapshot
+
+export const makeObjectKey = (sheet: ISheet, name: string) =>
+  `${sheet.address.sheetId}:${sheet.address.sheetInstanceId}:${name}`
```

Comment:
I was confused about the word "objectKey" as that's also the word we use in `@theatre/core` for `sheet.object(key, ...)`. Maybe rename it to `makeStoreKey()`?